main.css

```diff
@@ -0,0 +1,771 @@
+
+html,
+body {
+    font-family: sans-serif;
+    margin: 0px;
+}
+
+/* header */
+
+.header {
+    display: flex;
+    height: 220px;
+    border-bottom: 0.5px solid grey;
+    justify-content: center;
+    align-items: center;
+    gap: 40px
+}
+
+.header__glass{
+    position: relative;
+    bottom: 100px;
+    left: 40px;
+}
+
+.header__glass-source{
+    width: 35px;
+    height: 35px;
+    position: absolute;
+}
+
+.header__small-container{
```

Comment:
what is an element and what is a modificator here? small sounds like a modificator? doesn't it?